odm_sdk/scripts/internal/wait_for_host.py

```diff
@@ -36,19 +36,24 @@ def main():
         type=str, help='host address', required=True
     )
 
+    arg_parser.add_argument(
+        '--odm-endpoint', default='frontend/health',
+        type=str, help='(optional) specific endpoint to query', required=False
+    )
+
     args = arg_parser.parse_args()
 
     wait_cycle = 0
 
     while wait_cycle < ALLOWED_WAIT_CYCLES:
         time.sleep(TIMEOUT_IN_SECONDS)
-        if is_api_ready(args.odm_url):
-            print(f'Host {args.odm_url} is ready!')
+        if is_api_ready(args.odm_url, args.odm_endpoint):
```

Comment:
TBH I don't understand why do we need this adjustment now, since init container already interacts with ODM